src/main/java/com/spring/boot/util/CustomCsvLoaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.spring.boot.util;
+
+import java.io.BufferedInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URL;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CustomCsvLoaderImpl implements CustomCsvLoader {
+    @Override
+    public boolean loadCsvFile(String url, String intoPath) {
+        try (BufferedInputStream in = new BufferedInputStream(new URL(url).openStream());
```

Comment:
how much time does it take to load and parse the file on startup? (answer in slack please)